mne/io/open.py

```diff
@@ -255,7 +257,6 @@ def _show_tree(fid, tree, indent, level, read_limit, max_str):
                 out += [next_idt + prepend + str(k) + ' = ' +
                         '/'.join(this_type) + ' (' + str(size) + ')' +
                         postpend]
-                out[-1] = out[-1].replace('\n', u'¶')
```

Comment:
Yes I think so too. I wonder if it would be safest to stay as unicode now too so that if someone is expecting it be unicode (like if someone has show_fiff(..).encode('utf-8')) it won't break?